translations/es.ts

```diff
@@ -2349,13 +2349,13 @@ Si no estás seguro, selecciona &quot;No&quot; para evitar comunicaciones usando
     <message>
         <location filename="../src/widget/widget.cpp" line="497"/>
         <source>Toxcore failed to start, the application will terminate after you close this message.</source>
-        <translation>toxcore falló al iniciar, el programa terminará al cerrar este mensaje.</translation>
+        <translation>Toxcore falló al iniciar, el programa terminará al cerrar este mensaje.</translation>
```

Comment:
Then the original is wrong and needs correction.
Furthermore see here: https://wiki.tox.chat/doku.php  It writes "Compiling Toxcore", so apaprently it is not overly important th write toxcore strarting with a lowercase t. 
It looks extremely odd to have a sentence start in lower case. 
As long as the official wiki does not see a capital T as a problem, I'm inclined not to see it as such either.